scripts/genappimage.sh

```diff
@@ -34,8 +35,9 @@ make install
 
 # App version, used by generate_appimage.
 VERSION=$("$ROOT_DIR"/build/bin/nvim --version | head -n 1 | grep -o 'v.*')
+export VERSION
 
-cd "$APP_BUILD_DIR"
+cd "$APP_BUILD_DIR" || exit
```

Comment:
should we exit with error code here? same for the other case below

Reply:
It already does I think, `cd A || exit` is the same as `cd A || exit 1` as I understand it. Although it doesn't really hurt to be explicit so we might as well use `exit 1` for readability.

Reply:
oh I should have read `exit -help` : 

> Exits the shell with a status of N.  If N is omitted, the exit status is that of the last command executed.

That's definitely better. So it's good as you have it.